docs/products/opensearch/howto/opensearch-log-integration.rst

```diff
@@ -12,23 +12,33 @@ Here are the steps needed to enable logs integration. This allows you to send lo
 
 1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
 
-2. On the **Services** page, open the service you want to send the logs from.
+2. On the **Services** page, scroll down and choose **Service integrations**.
 
-3. On the **Logs** tab, click on **Enable Logs integration**. 
+3. On the **Logs** tab, click on **Use Integration**. 
 
-4. Choose an existing OpenSearch instance or create a new one, then select **Continue**.
+4. Select an existing OpenSearch instance or create a new one, then select **Continue**.
 
-5. Configure your ``index prefix`` and ``retention limit`` parameters, then select **Enable**
+5. Configure your ``index prefix`` and ``index retention limit`` parameters, then select **Enable**
 
 .. note::
-    If you want to effectively disable the index ``retention limit``, you can set it to the maximum value which is 10000 days.
+    If you want to effectively disable the index ``index retention limit``, you can set it to the maximum value which is 10000 days.
 
 Your Aiven service is now sending logs to your OpenSearch service which you can explore further.
 
+Logs integration configuration
+------------------------------
+There are two parameters that you can adjust when integrating logs to your OpenSearch service:
+
+* ``index prefix``, specifies the prefix part.
+* ``index retention limit``, number of days to preserve the daily indexes.
+
+.. warning::
+    When the ``index retention limit`` is reached, the daily indexes are deleted. The count starts from the day that those indexes were sent.
```

Comment:
I'm not sure I understand what means "The count starts from the day that those indexes were sent." , is  "were sent" means "were created"? Would be helpful to be more specific and clear here